Meden-Agan-27/travel-worms
# Features and style

What I did:
- when you are in a book page and you want to ad that book on a new bookshelf, when you create the new shelf, the book is also added
- you can change the status of a book 
How to test:
- pick a book, from books, on book page, 'add to bookshelf', 'create a new bookshelf' and you should have the chosen book in your new created shelf
 - now, try to change the status of your book by selecting a new status from the dropdownlist in the bookshelf
